client/app/bundles/course/assessment/containers/AssessmentForm/translations.intl.js

```diff
@@ -99,16 +99,26 @@ const translations = defineMessages({
     id: 'course.assessment.form.passwordProtection',
     defaultMessage: 'Password Protection',
   },
-  passwordProtectionHint: {
-    id: 'course.assessment.form.passwordProtectionHint',
-    defaultMessage: "When password protection is enabled, students are allowed to access their \
+  viewPasswordHint: {
+    id: 'course.assessment.form.viewPasswordHint',
+    defaultMessage: 'When assessment password is enabled, students are required to input the password in order to \
+      view/attmpet the assessment.',
```

Comment:
'attempt'

Reply:
I don't think each unique time slot gets a different password, this is a single password for attempting the entire assessment.

Reply:
For this, since there are multiple ways of using it, suggest we just be factual and let users figure out how to use it.